mycroft/tts/tts.py

```diff
@@ -475,6 +497,7 @@ def load_tts_plugin(module_name):
 
 
 class TTSFactory:
+    """Factory class instanciating the configured TTS engine."""
```

Comment:
engines?

Reply:
It only instantiates a single engine. Added some text on the engines available.